cdc/kv/region_worker.go

```diff
@@ -725,7 +720,7 @@ func (w *regionWorker) handleEventEntry(
 					zap.Uint64("regionID", regionID))
 				return errUnreachable
 			}
-			ok := state.matcher.matchRow(entry)
+			ok := state.matcher.matchRow(entry, state.initialized)
 			if !ok {
 				if !state.initialized {
 					state.matcher.cacheCommitRow(entry)
```

Comment:
Seems `entry` can be a prewrite without value?

Reply:
It's alway a COMMIT, see line 718.